recipes/update_checker/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "update_checker" %}
+{% set version = "0.9" %}
+{% set md5 = "6d8ee1de9b67741fc394f7cf163ea658" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  md5: {{ md5 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - requests >=1.0.2
+    - setuptools
+
+  run:
+    - python
+    - requests >=1.0.2
+    - setuptools
```

Comment:
Does this really require `setuptools` at run time? It doesn't look like it should. 😕